src/geometry.rs

```diff
@@ -487,6 +533,81 @@ pub fn cube_sharp_same_len(position: [f32; 3], scale: f32) -> Geometry {
     Geometry::from_triangle_faces_with_vertices_and_normals(faces, vertex_positions, vertex_normals)
 }
 
+#[allow(dead_code)]
+pub fn cube_sharp_same_len_welded(position: [f32; 3], scale: f32) -> Geometry {
```

Comment:
This is definitely not `same_len` 😉

Reply:
What is the benefit of this vs `cube_sharp_var_len()`?

Reply:
> This is definitely not `same_len` 😉

it wasn't clear what `same_len` means in this context. i can rename it

Reply:
> What is the benefit of this vs `cube_sharp_var_len()`?

sharp normals, edgy shading. no benefits, just correct geometry